doc/HOWTO_related_repos.md

```diff
@@ -0,0 +1,115 @@
+1. Run [`bin/merge_describer`](bin/merge_describer) which generates a helpful description you can copy/paste into your PR.  It contains the commit messages and links to the merged branches since the last merge to master.  You can also pass in other branch names or tags to target for the comparison.
+1. Push up your new branch to GitHub, and paste the output from `bin/merge_describer` into the PR description.  For clarity it's best to sort commits into 3 groups: features, fixes, and tech tasks.
+
+### Notes
+
+You can use the [`bin/latest_from_open`](bin/latest_from_open) script to create a new `latest_from_open_MMDDYYYY` branch instead of using the steps above.  Add the `upstream` remote, then run `bin/latest_from_open 04012021`.
+
+When merging in changes from the open source repo, differences in `scehma.rb` and `Gemfile.lock` tend to be the trickiest to resolve.  Run `bundle` and `rails db:migrate` locally to ensure any conflicts have been resolved as expected.
+
+## Squash vs Merge
+
+On github, squash commits include a parenthesized branch number like `(#45)` in the commit message by default, except when the PR only includes 1 commit. Merge commits do not include the branch number.
+The `bin/merge_describer` script only describes commits which include the parenthesized branch number,
+so use Squash (and make sure the parenthesized branch number is in the commit message) for every PR that should be described later on in the GitHub release and release ticket.  Use Merge commits for `latest_from_open_MMDDYYYY` branch PRs.
+
+For feature PRs, the PR description will become the squash commit message. These appear later in the `bin/merge_describer` output, which is used in the`latest_from_open_MMDDYYYY` PR description, github release, and release redmine ticket. Try to write feature PR descriptions that focus on user outcomes rather than technical details - "Fix ability to download CSV" is preferred over "Resolve issues with JSON parsing".
+
+## Creating a GitHub release
+
+1. Check out the commit you want to release and tag it: `git tag v2021-02-18`
+1. Use `bin/merge_describer v2021-01-28` to list all commits since the previous release tag
+1. Organize the output into features, fixes and tech tasks
+1. Create a new release in Github using this list of commits for the description
+1. Specify the new tag (`v2021-02-18`) for the release tag version, target, and release title
+
+# Bringing changes from your school-specific repo into `nucore-open`
+
+_While in the `nucore-open` directory_
+
+For example, to bring changes from `nucore-nu` into `nucore-open`:
+
+`git remote add nu git@github.com:tablexi/nucore-nu.git`
+
+The best way to bring in the changes from NU is by cherry-picking the individual
+commits. For this reason, it's best to develop on `nucore-open` to start with, or to squash
+the commits in your branch so you only need to bring one commit.
+
+```
+git fetch nu
+git cherry-pick XXXXX
+```
+
+# Deploy
+
```

Comment:
If there is a preferred deployment method? "Method may vary depending on which school"